pyiron_base/project/generic.py

```diff
@@ -1510,8 +1508,24 @@ def unpack(self, origin_path, csv_file_name='export.csv', compress=True):
 class Maintenance:
     """
     The purpose of maintenance class is to provide
-    some measures of perfomance for pyiron
+    some measures of perfomance for pyiron, whether local to the project
+    or global (describing the status of pyiron on the running machine)
     """
+    def __init__(self):
+        """
+        initialize the local and global attributes to None
+        """
+        self._global = None
+        self._local = None
+
+    @property
+    def global_status(self):
+        if self._global is None:
+            self._global = GlobalMaintenance()
+            return self._global
```

Comment:
I would just name the property `global` and I tend to init all Maintenance classes at once if someone is already accessing the maintenance: 
```suggestion
    def __init__(self):
        """
        initialize the local and global attributes
        """
        self._global = GlobalMaintenance()
        self._local = None

    @property
    def global(self):
        return self._global
```